crates/eth-rpc/Cargo.toml

```diff
@@ -37,8 +37,8 @@ starknet = { workspace = true }
 thiserror = "1.0.38"
 tower = "0.4.13"
 tower-http = "0.4.1"
-tracing = "0.1.34"
-tracing-subscriber = { version = "0.3.3", features = ["env-filter"] }
```

Comment:
why moving the feature from crate to workspace? see also https://doc.rust-lang.org/nightly/cargo/reference/workspaces.html#the-dependencies-table

Reply:
So we can inherit it as we're using it for the ctor as well in [hive-utils](https://github.com/kkrt-labs/kakarot-rpc/blob/aadf3b3fba77957c22ce8558ec6940e12310cae9/crates/hive-utils/src/madara/utils.rs#L183)